source/renderer/app/api/network/types.js

```diff
@@ -68,6 +73,13 @@ export type NetworkInfoResponse = {
   },
 };
 
+export type NetworkClockResponse = {
+  status: 'available' | 'unavailable',
```

Comment:
@yakovkaravelov as per documentation the status can also be `pending`: https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/getNetworkClock
Please update the type definition. Thanks!